examples/demo/src/products/GridList.js

```diff
@@ -18,7 +18,7 @@ const styles = {
             'linear-gradient(to top, rgba(0,0,0,0.8) 0%,rgba(0,0,0,0.4) 70%,rgba(0,0,0,0) 100%)',
     },
     link: {
-        color: 'white',
+        color: '#fff!important',
```

Comment:
:scream:

Reply:
yeah, I know. If you find a better way, I'm your man.

Reply:
Well, it works without `!important` on chrome

Reply:
not on my laptop with the latest material ui

Reply:
I used this branch and did not forget to `make install`